src/classes/HealthCheckVMapper.cls

```diff
@@ -88,25 +88,31 @@ public virtual with sharing class HealthCheckVMapper {
     * @description Retrieve a view model representing a health check to run.
     * @return A HealthCheckVModel representing details of a health check to be run.
     **********************************************************************************/
-    public virtual HealthCheckVModel getHealthCheckViewModel(){
-        FeatureParameterDateModel featureParamDateModel = 
-           (FeatureParameterDateModel)this.locateFeatureParameterService().getFeatureParameterModel();
-        String lastRunDate = featureParamDateModel.convertToString(); 
+    public virtual HealthCheckVModel getHealthCheckViewModel() {
```

Comment:
I may have missed this in a previous review. This class uses a number of tabs. Please fix the spacing issues to use four spaces.